src/textual/widgets/_tabbed_content.py

```diff
@@ -151,37 +196,37 @@ def validate_active(self, active: str) -> str:
             Value of `active`.
 
         Raises:
-            ValueError: If the active attribute is set to empty string.
+            ValueError: If the active attribute is set to empty string when there are tabs available.
         """
-        if not active:
-            raise ValueError("'active' tab must not be empty string.")
+        if not active and self.get_child_by_type(ContentSwitcher).current:
```

Comment:
Is there any chance the dev might attempt to set `active` earlier, and there won't be a `ContentSwitcer` yet?

Reply:
I *think* the only way that could arise is if they do that before the widget is composed, but even without this change that would end up with a similar issue due to the triggering of `_watch_active`, right? Attempting to set the active tab to a pane that doesn't even exist yet is never going to go well.